sources/osg/Light.js

```diff
@@ -305,6 +305,15 @@ define( [
             return this._ground[ 3 ] >= 0.0;
         },
 
+        getPositionnedMatrix: function () {
```

Comment:
those methods should be removed, there are not consistent in other usecase than shadow